clients/besu/Dockerfile

```diff
@@ -17,11 +17,7 @@ ADD genesis.json /genesis.json
 ADD enode.sh /enode.sh
 RUN chmod +x /enode.sh
 
-RUN echo "{\
-  \"repo\":\"https://github.com/hyperledger/besu\",\
-  \"branch\":\"${branch:-detached}\", \
-  \"commit\":\"${VCS_REF:-xxxxxx}\"\
-}" > /version.json
+RUN echo "repo: https://github.com/hyperledger/besu, branch: "${branch:-detached}", commit:"${VCS_REF:-xxxxxx}"" > /version.txt
```

Comment:
We can remove the repo URL here, and try to add a version number instead.

Reply:
Can I just shovel output from `<client> --version` > /version.txt ? @fjl